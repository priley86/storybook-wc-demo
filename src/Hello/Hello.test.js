/* eslint-env jest */
/* 
  JS Dom Web Component Polyfill Workaround for Jest:
  https://github.com/tmpvar/jsdom/issues/1030

  Will be resolved in: 
  https://github.com/tmpvar/jsdom/pull/1872
*/
import 'document-register-element'

import React from 'react'
import renderer from 'react-test-renderer'

import { Hello } from '../index.js'

test('Hello renders default properly', () => {
  const component = renderer.create(<Hello />)

  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})

test('Hello renders name properly', () => {
  const component = renderer.create(<Hello name="Bob" />)

  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
