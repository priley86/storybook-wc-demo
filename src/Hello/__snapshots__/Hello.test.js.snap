// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hello renders default properly 1`] = `
<pf-hello
  name="I Love Web Components!"
/>
`;

exports[`Hello renders name properly 1`] = `
<pf-hello
  name="Bob"
/>
`;
